### PR TITLE
fix deathsquads dying to bombs

### PR DESCRIPTION
**Pull request by k-k-karma**
armor values aren't capped at 100 i dont think lol

### CODE REVIEW

**Comment by yoyobatty** [comment]
Nigga that's not how the CODE WORKS!

**Comment by yoyobatty** [comment]
`	
var/hit_percent = (100-blocked)/100
where blocked = 1233243124223400 
100 - 1233243124223400 = -1233243124223300= hit_percent
if(!damage || (!forced && hit_percent <= 0))
	return 0
var/damage_amount =  forced ? damage : damage * hit_percent
plug that in
damage = 120 lets say
hit percent = -1233243124223300
120 * -1233243124223300 = -147989174906796000
In the code this proc tells to adjust brute loss
adjustBruteLoss(damage_amount, forced = forced)
So basically you're adjusting -large number in brute loss which in turn heals you.
`

**Comment by Manly-Dwarf** [comment]
that code better not work lol

**Comment by k-k-karma** [comment]
do they die to bombs?

**Comment by HideAndSeekLOGIC** [comment]
@k-k-karma what is even the issue? where does the deathsquad have to be to die to the bombs? I can't even understand it because you refuse to give any sort of description

**Comment by k-k-karma** [comment]
> @k-k-karma what is even the issue? where does the deathsquad have to be to die to the bombs? I can't even understand it because you refuse to give any sort of description

the issue is that they aren't immune to bombs,

**Comment by HideAndSeekLOGIC** [comment]
>the issue is that they aren't immune to bombs,

**Comment by k-k-karma** [comment]
don't close PRs to make a point, dipshit

**Comment by k-k-karma** [comment]
wrong button

**Comment by HideAndSeekLOGIC** [comment]
Either karma is genuinely retarded or this PR is a literal joke.

**Comment by HideAndSeekLOGIC** [comment]
Do not merge this PR till karma does tests

**Comment by yoyobatty** [comment]
Stale.
